packages/appium/test/e2e/cli-driver.e2e.spec.js

```diff
@@ -111,41 +111,43 @@ describe('Driver CLI', function () {
     });
 
     it('should show updates for installed drivers with --updates', async function () {
-      if (system.isWindows()) {
-        // TODO figure out why this isn't working on windows
-        return this.skip();
-      }
-      const versions = /** @type {string[]} */ (
-        JSON.parse(
-          (
-            await exec('npm', ['view', '@appium/fake-driver', 'versions', '--json'], {
-              encoding: 'utf-8',
-            })
-          ).stdout
-        )
-      );
-
-      const penultimateFakeDriverVersionAsOfRightNow = versions[versions.length - 2];
-
-      await resetAppiumHome();
-      await runInstall([
-        `@appium/fake-driver@${penultimateFakeDriverVersionAsOfRightNow}`,
-        '--source',
-        'npm',
-      ]);
-      const {fake} =
-        /** @type {Record<string,import('appium/lib/cli/extension-command').InstalledExtensionListData>} */ (
-          await runList(['--updates'])
-        );
-      const updateVersion = String(fake.updateVersion || fake.unsafeUpdateVersion);
-      util.compareVersions(
-        String(updateVersion),
-        '>',
-        penultimateFakeDriverVersionAsOfRightNow
-      ).should.be.true;
-      // TODO: this could probably be replaced by looking at updateVersion in the JSON
-      const {stderr} = await runAppiumRaw(appiumHome, [DRIVER_TYPE, LIST, '--updates'], {});
-      stderr.should.match(new RegExp(`fake.+[${updateVersion} available]`));
+      // FIXME turn back on when fake-driver supports appium 3 beta
```

Comment:
would it help if we just hack it as
```
if (process.env.CI) {
  return this.skip();
}
```

so the rest of the test shouldn't be commented out?

Reply:
but the tests will also fail locally...

Reply:
I assume it is ok if we add a proper explanation comment %)

Reply:
ok, I published an appium server version and a new fake driver, and unquarantined the tests so all the tests but one pass (that test downgrades fake driver and then updates it, which doesn't work because we would need multiple published versions of fake driver that support appium 3 🙄 ). so that one test is quarantined and we'll need to update after the next server version publish i guess.